### PR TITLE
Added Stack container

### PR DESCRIPTION
**Pull request by diegolanda**
Added Stack layout

stacklayout

### CODE REVIEW

**Review by raeioul — APPROVED** [review]

File: src/components/Layouts/Stack.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  getFlow = () => {
+    const { orientation } = this.props;
+
+    return orientation === ORIENTATION.vertical ? 'column' : 'row' ;
+  }
+
+  render() {
+    const { children } = this.props;
+
+    const flow = this.getFlow();
```
Comment (oscarfgq) [inline src/components/Layouts/Stack.js]: ¿Por qué no hacer aquí directamente `orientation === ORIENTATION.vertical ? 'column' : 'row' ;`?

**Review by hdwilber — APPROVED** [review]
LGTM
Perhaps, the names of HTML classes should be only lowercase, `wui-stack`, or `wuiStack`.
Anyway, it looks good.

File: src/components/Layouts/__tests__/Stack.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import Stack, { ORIENTATION } from '../Stack';
+
+import { mount } from 'enzyme';
+import { wrap } from 'module';
```
Comment (oscarfgq) [inline src/components/Layouts/__tests__/Stack.js]: Parece que no se está usando.

**Comment by Taxxx** [comment]
Nice dude @diegolanda

**Comment by m-camacho** [comment]
css class names should be in kebab-case as standards... but I don't have a link to the standards :v

File: src/components/Layouts/Stack.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.wui-Stack {
+  display: flex;
+}
```
Comment (m-camacho) [inline src/components/Layouts/Stack.css:3]: Please add an empty line at the end of each file 👍

**Comment by diegolanda** [comment]
@m-camacho can we vote on #2 ?

File: src/components/Layouts/Stack.js (hunk 1 above)
Comment (oscarfgq) [reply src/components/Layouts/Stack.js]: ¿Why not doing it here directly: `orientation === ORIENTATION.vertical ? 'column' : 'row' ;`?

File: src/components/Layouts/__tests__/Stack.js (hunk 2 above)
Comment (oscarfgq) [reply src/components/Layouts/__tests__/Stack.js]: Seems that this is not being used.

File: src/components/Layouts/Stack.css (hunk 3 above)
Comment (m-camacho) [reply src/components/Layouts/Stack.css:3]: I would prefer to rename the file to `stack.css` instead. But it is just based in my previous projects... there is no other reason.

File: src/components/Layouts/Stack.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+export const ORIENTATION = {
+  vertical: 'vertical',
+  horizontal: 'horizontal'
+};
+
+class Stack extends PureComponent {
+  getFlow = () => {
+    const { orientation } = this.props;
+
+    return orientation === ORIENTATION.vertical ? 'column' : 'row' ;
```
Comment (m-camacho) [inline src/components/Layouts/Stack.js:15]: in next PRs we can move hardcoded strings to constants.

File: src/components/Layouts/Stack.js (hunk 1 above)
Comment (oscarfgq) [inline src/components/Layouts/Stack.js:21]: Why not doing it here directly: `orientation === ORIENTATION.vertical ? 'column' : 'row' ;`?

File: src/components/Layouts/__tests__/Stack.js (hunk 2 above)
Comment (oscarfgq) [inline src/components/Layouts/__tests__/Stack.js]: Seems that this is not being used.

**Review by m-camacho — APPROVED** [review]
A couple of minor things.
Code LGTM 👍

**Comment by diegolanda** [comment]
@oscarfgq  please review again,

The idea behind `getFlow` is to not overload visually `render` of stuff that may not be strictly related

File: src/components/Layouts/Stack.js (hunk 1 above)
Comment (diegolanda) [reply src/components/Layouts/Stack.js:21]: The idea is not to overload render with lines not strictly related

File: src/components/Layouts/__tests__/Stack.js (hunk 2 above)
Comment (diegolanda) [reply src/components/Layouts/__tests__/Stack.js]: Good catch

**Pull request merged**